.github/ISSUE_TEMPLATE/project_story_template.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Project Story Template
+about: Project story template for tracking on the infra project board 
+title: ''
+labels: card
```

Comment:
I don't think we need this label either.

Reply:
Can remove. Was copying the civic one